docs/contents/installation.md

```diff
@@ -0,0 +1,23 @@
+# Installing dmenv
+
+## Using the pre-compiled binaries
+
+The easiest way is to download the matching binary from the [releases page](https://github.com/TankerHQ/dmenv/releases) for your platform and put it
+somewhere on in your $PATH:
+
+### Linux, macOS
+
+```console
+cd ~/.local/bin
+curl-L https://github.com/TankerHQ/dmenv/releases/download/v<version>/dmenv-<platform> -o dmenv
+chmod u+x dmenv
+```
+Note: replace $PLATFORM by your current platform: `linux`, `osx` or `windows`.
+
+### Windows
+
+Download the `dmenv-windows.exe` file and save it for instance in `c:\path\to\python\Scripts\dmenv.exe`.
```

Comment:
... file **from the [releases](https://github.com/TankerHQ/dmenv/releases) page** and save...